cmd/bridge/main.go

```diff
@@ -418,6 +429,15 @@ func main() {
 
 		srv.TerminalProxyTLSConfig = serviceProxyTLSConfig
 
+		if *fK8sModeOffGitOps != "" {
+			offClusterGitOpsURL := bridge.ValidateFlagIsURL("k8s-mode-off-gitops", *fK8sModeOffGitOps)
```

Comment:
So previously you were adding `https` here like `bridge.ValidateFlagIsURL("k8s-mode-off-gitops", "https://"+*fK8sModeOffGitOps)`.

Reply:
`https` is added for both [off-cluster ](https://github.com/chetan-rns/console/blob/d6a1ed94ad749289166787c060286de0ce2766f6/contrib/oc-environment.sh#L37)and [in-cluster](https://github.com/chetan-rns/console/blob/d6a1ed94ad749289166787c060286de0ce2766f6/cmd/bridge/main.go#L367) scenarios.